src/config.h.in

```diff
@@ -106,21 +99,12 @@
 /* Define to 1 if you have the <limits.h> header file. */
 #undef HAVE_LIMITS_H
 
-/* Define to 1 if you have the `lockf' function. */
-#undef HAVE_LOCKF
```

Comment:
HAVE_LOCKF is checked in [adv-flock.c](https://github.com/macports/macports-base/blob/master/src/pextlib1.0/adv-flock.c#L68).

Reply:
this was an auto-gen change - forgot to check for defines :\ - thoughts on just removing the HAVE check?

Reply:
Probably fine if lockf is going to exist on any reasonable target platform. It's used as a fallback if flock isn't available.